pkg/credentialprovider/azure_credentials.go

```diff
@@ -232,24 +240,55 @@ func (a *acrProvider) getFromACR(ctx context.Context, loginServer string) (strin
 // parseACRLoginServerFromImage takes image as parameter and returns login server of it.
 // Parameter `image` is expected in following format: foo.azurecr.io/bar/imageName:version
 // If the provided image is not an acr image, this function will return an empty string.
-func (a *acrProvider) parseACRLoginServerFromImage(image string) string {
-	match := acrRE.FindAllString(image, -1)
+func (a *acrProvider) parseACRLoginServerFromImage(image string) (string, string) {
+	processedImage, mirrorRegistry := a.processImageWithMirrorMapping(image)
```

Comment:
the second returned value is actually the source registry or the original registry. Right? I feel that `mirrorRegistry` is a little confusing.

Reply:
Good idea. Now I use `source` and `target` for all registry/image var name.